lib/Mojolicious/Plugin/PODViewer.pm

```diff
@@ -208,10 +215,16 @@ sub _html {
 
   # Rewrite code blocks for syntax highlighting and correct indentation
   for my $e ($dom->find('pre > code')->each) {
+    my $attrs = $e->parent->attr;
+    my $class = $attrs->{class};
+    my $parent_class = 'codesample rounded';
+    $attrs->{class} = defined $class ? "$class $parent_class" : $parent_class;
```

Comment:
This block of changes seems to be adding `codesample rounded` classes to all `<pre><code>...</code></pre>` in the HTML, but we don't need classes when we can select by the elements. The class below (`prettyprint`) only applies when we detect Perl code that isn't a `Usage` (like from a Mojolicious command). Can we do without this change?

Reply:
The idea is to add a background color to all <code> blocks. If the user doesn't like it, he can ignore this class in his stylesheet.